python/modules/IcePy/ValueFactoryManager.cpp

```diff
@@ -29,7 +26,7 @@ namespace
     }
 }
 
-/* static */ ValueFactoryManagerPtr
```

Comment:
That's our convention for static function implementations.

Reply:
There is only this one and the one in ruby.

```
ice % rg "/\* static \*/"
ruby/src/IceRuby/ValueFactoryManager.cpp
51:/* static */ ValueFactoryManagerPtr
jose@studio ice % 
```

Reply:
ok, looks like we dropped this convention!